qemu/tests/netkvm_protocol_binding.py

```diff
@@ -58,9 +66,19 @@ def run(test, params, env):
 
     test.log.info("Will install inf file found at '%s'", inf_path)
     install_cmd = params["install_cmd"] % inf_path
-    status, output = session.cmd_status_output(install_cmd, timeout=timeout)
-    if status:
-        test.error("Install inf file failed, output=%s" % output)
+
+    # workaround for driver no signature
+    key_to_install_driver = params.get("key_to_install_driver").split(";")
+    thread_installation = utils_misc.InterruptedThread(session.cmd_status_output, (install_cmd, timeout))
+    thread_installation.start()
+    time.sleep(10)
+    thread_sendkey = utils_misc.InterruptedThread(send_key, (vm, key_to_install_driver))
+    thread_sendkey.start()
+    try:
+        thread_installation.join()
+    except Exception as err:
+        err_msg = "Error Info: '%s'"
+        test.error(err_msg % err)
```

Comment:
I don't see the necessity for this Exception handling as no real extra operations or information.

Reply:
Not sure if I really understand what you mean. The code has been updated, could you help review it again. Thanks!